src/glbackend/GLAbstraction/vertexarray.jl

```diff
@@ -40,7 +40,10 @@ function attach2vao(buffers, attrib_location, kind)
 end
 
 @enum VaoKind simple elements elements_instanced
-
+#TODO speedup: Maybe it would be better for performance making our vertex arrays
+#              primitives, i.e. making the buffers an NTuple instead of a vector.
+#              I think that should be possible since nobody really wants to change
+#              the vao after it's made anyway?
```

Comment:
sounds good! shouldn't change performance much though, since we just access the id!